### PR TITLE
Renable license checks

### PR DESCRIPTION
**Pull request by RubenRBS**
Some files need some discussion before merging

### CODE REVIEW

File: lib/compilers/typescript.js [hunk 1]
```diff
@@ -1,3 +1,27 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```
Comment (RubenRBS) [inline lib/compilers/typescript.js:1]: @ODuzhar the file you contributed 2 months ago was missing a license. Are you ok with this copyright attribution?

File: static/panes/gccdump-view.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
-// Copyright (c) 2017, Marc Poulhiès - Kalray Inc.
-// Copyright (c) 2022, Compiler Explorer Authors
+// Copyright (c) 2017, Marc Poulhiès - Kalray Inc, 2022, Compiler Explorer Authors
```
Comment (RubenRBS) [inline static/panes/gccdump-view.ts:1]: With the old way, the linter was complaining about a bad header. Is this format ok?

File: test/pp-output-cases/filter-tests.js [hunk 3]
```diff
@@ -1,3 +1,27 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```
Comment (RubenRBS) [inline test/pp-output-cases/filter-tests.js:1]: @jeremy-rifkin this file you contributed 2 months ago was missing a license header. Are you ok with this copyright attribution?

File: test/unfurl-tests.js [hunk 4]
```diff
@@ -1,3 +1,27 @@
+// Copyright (c) 2022, Compiler Explorer Authors
```
Comment (RubenRBS) [inline test/unfurl-tests.js:1]: File first commited in 2021

File: test/pp-output-cases/filter-tests.js (hunk 3 above)
Comment (jeremy-rifkin) [reply test/pp-output-cases/filter-tests.js:1]: Yes thank you!

File: test/unfurl-tests.js (hunk 4 above)
Comment (matsjla) [reply test/unfurl-tests.js]: I think it'd be correct to change it to 2021 then.

File: lib/compilers/typescript.js (hunk 1 above)
Comment (ASDAlexander77) [reply lib/compilers/typescript.js:1]: sure

**Review by matsjla — APPROVED** [review]

File: static/panes/gccdump-view.ts (hunk 2 above)
Comment (dkm) [reply static/panes/gccdump-view.ts:1]: I've never seen it like this. The lint wants a unique copyright notice?

Comment (RubenRBS) [reply static/panes/gccdump-view.ts:1]: The pattern it's looking for is `^ Copyright \\(c\\) \\d{4}, .*$` which I'd think works for two headers, but somehow it must be making sure that the template is also being followed, which trips the linter here. No idea what to do

**Comment by RubenRBS** [comment]
After fighting the linter on this one, I'm going to merge the missing license headers and try to fix whatever is causing the rule to always trigger even on seemingly ok headers

**Pull request merged**